vnext/template/cs-lib/proj/MyLib.csproj

```diff
@@ -25,6 +25,9 @@
     <UseExperimentalNuget>true</UseExperimentalNuget>    <!-- This will eventually default to true and this property should be removed -->
     {{/useExperimentalNuget}}
   </PropertyGroup>
+  <PropertyGroup Label="NuGet">
+    <AssetTargetFallback>$(AssetTargetFallback);native</AssetTargetFallback>
+  </PropertyGroup>
```

Comment:
This template change will not get copied over to existing libraries/community modules. Do we need it for compatibility with existing 3p community modules?

Reply:
If the current community modules still use `packages.config`, then no.